demo.ipynb

```diff
@@ -18,12 +18,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 39,
```

Comment:
It looks like the notebook was modified for this PR.  Maybe an errant `git commit -a`